lxd/container.go

```diff
@@ -397,7 +399,7 @@ func containerValidDevices(db *db.Node, devices types.Devices, profile bool, exp
 				return fmt.Errorf("Unix device entry is missing the required \"source\" or \"path\" property.")
 			}
 
-			if m["major"] == "" || m["minor"] == "" {
+			if m["required"] == "" || (shared.IsTrue(m["required"]) && (m["major"] == "" || m["minor"] == "")) {
```

Comment:
Hmm, is that condition right?

Shouldn't it instead be:

    if (m["required"] == "" || shared.IsTrue(m["required"])) && (m["major"] == "" || m["minor"] == "") {

Reply:
addressed